src/renderer/widgets/PasswordInputField.cpp

```diff
@@ -199,7 +200,7 @@ bool CPasswordInputField::draw(const SRenderData& data) {
             forceReload = true;
     }
 
-    return dots.currentAmount != PASSLEN || data.opacity < 1.0 || forceReload;
+    return dots.currentAmount != PASSLEN || fade.animated || forceReload;
```

Comment:
`data.opacity < 1.0 ` did not really make sense to me. Did I miss something about that?

Reply:
uhh I think the renderer will take care of that anyways I think